src/HtmlUtils.tsx

```diff
@@ -50,7 +50,7 @@ const SURROGATE_PAIR_PATTERN = /([\ud800-\udbff])([\udc00-\udfff])/;
 const SYMBOL_PATTERN = /([\u2100-\u2bff])/;
 
 // Regex pattern for Zero-Width joiner unicode characters
-const ZWJ_REGEX = /[\u200D\u2003]/g;
+const ZWJ_REGEX = /[\u200D\u2003\u200B]/g;
```

Comment:
This isn't quite correct, since a zero-width space isn't a zero-width joiner; I'm also not sure what U+2003 EM SPACE is doing in there either, especially since it's already covered by the `\s` character class in `WHITESPACE_REGEX`. Since both `WHITESPACE_REGEX` and `ZWJ_REGEX` are used for the exact same purpose in this file and nowhere else, how about combining both of those into `EMOJI_SEPARATOR_REGEX` instead?